templates/taxi/car_detail.html

```diff
@@ -19,4 +19,19 @@ <h1>Drivers</h1>
       <li>{{ driver.username }} ({{ driver.first_name }} {{ driver.last_name }})</li>
     {% endfor %}
   </ul>
+  
+  <hr>
+  {% if user in car.drivers.all %}
+    <form method="post" action="{% url 'taxi:car-detail' pk=car.id %}">
```

Comment:
this can be one form - use condition to show assign/remove inputs
Also you can leave action empty since you post to the same view

Reply:
One form is enough here - something like this:
```
<form method="post">
    {% csrf_token %}
    {{ form }}
    {% if user in car.drivers.all %}
    <input type="submit" value="remove">
    {% else %}
    <input type="submit" value="assign">
    {% endif %}
  </form>